Cargo.toml

```diff
@@ -1,14 +1,17 @@
 [package]
 name = "k8s-controller"
-version = "0.1.0"
+version = "0.2.0"
 authors = ["Matt Butcher <matt.butcher@microsoft.com>"]
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-kube = "0.12.0"
+kube = "0.24.0"
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
 failure = "0.1.5"
+futures = "0.3.1"
+anyhow = "1.0.26"
+tokio = { version = "0.2.6", features = ["full"] }
```

Comment:
@technosophos: After upgrading kube, I adjusted the code to follow the pattern in an [example program](https://github.com/clux/kube-rs/blob/ec2a87bfd3a847804e88d8f1512bbe63117a8e88/examples/node_informer.rs), but I still don't understand why `anyhow` and `tokio` need to be here rather than in `[dev-dependencies]` as in kube's [Cargo.toml](https://github.com/clux/kube-rs/blob/ec2a87bfd3a847804e88d8f1512bbe63117a8e88/Cargo.toml#L46-L47). Or perhaps it's more surprising that the kube examples don't need runtime dependencies on these crates.

Reply:
Tokio needs to be there to import the `tokio` macros. And I believe `anyhow` is there because of the traits (probably a `From` trait.) I don't know why Kube doesn't need them.